package.json

```diff
@@ -9,14 +9,18 @@
     "web": "expo start --web"
   },
   "dependencies": {
+    "@react-native-async-storage/async-storage": "^1.22.3",
+    "@react-navigation/native-stack": "^6.9.22",
     "expo": "50.0.2",
     "expo-status-bar": "1.11.1",
-    "react": "18.2.0",
-    "react-native": "0.73.2"
+    "react-native": "0.73.2",
+    "react-native-dropdown-select-list": "^2.0.5",
+    "react-native-modern-datepicker": "^1.0.0-beta.91"
   },
   "devDependencies": {
-    "@babel/core": "7.20.0",
+    "@babel/core": "7.20.2",
```

Comment:
This should be it's on PR but lets just do it anyway, because its causing us problems